lib/discourse/version.rb

```diff
@@ -4,17 +4,15 @@ module Discourse
   VERSION_REGEXP ||= /\A\d+\.\d+\.\d+(\.beta\d+)?\z/
```

Comment:
I think this file should definitely go somewhere in the app directory as it is app related.